### PR TITLE
byo performance_regression_test with gce

### PR DESCRIPTION
**Pull request by abvgedeika**
add update_db_packages feature for gce perf tests:
before the performance scenarios update the cluster with the packages located in update_db_packages folder(yaml conf parameter)

### CODE REVIEW

File: sdcm/cluster.py [hunk 1]
```diff
@@ -3008,6 +3012,7 @@ def node_setup(node, seed_address):
                 pass
 
         self.update_db_binary(node_list)
+        self.update_db_packages(node_list)
```
Comment (kongove) [inline sdcm/cluster.py]: Yes, it's lost in GCE backend. We need this

File: sdcm/cluster.py [hunk 2]
```diff
@@ -516,7 +517,10 @@ def _register_grafana_json(json_filename, url):
         json_mapping = {'scylla-data-source.json': 'datasources',
                         'scylla-dash-timeout-metrics.json': 'dashboards/db'}
 
-        process.run('sudo yum install git -y')
+        if platform.linux_distribution()[0].lower() == 'ubuntu':
+            process.run('sudo apt-get --assume-yes install git')
+        else:
+            process.run('sudo yum install git -y')
```
Comment (kongove) [inline sdcm/cluster.py]: We don't have a good support for running sct in ubuntu, it's not enough to just fix this.
This fix should be split to another patch.

Comment (abvgedeika) [inline sdcm/cluster.py]: In my runs I need only these changes on ubuntu)

Comment (kongove) [inline sdcm/cluster.py]: Ok

**Comment by abvgedeika** [comment]
split on https://github.com/scylladb/scylla-cluster-tests/pull/278 & https://github.com/scylladb/scylla-cluster-tests/pull/279

**Comment by kongove** [comment]
Hi @abvgedeika, one PR can contain multiple patches. The two patches changed same file and not unrelated, so it's ok to put them in one PR.

I already review and merged the PR #278 & #279
